src/node_lib.h

```diff
@@ -103,7 +103,7 @@ NODE_EXTERN void Initialize(const std::string& program_name = "node_lib",
  * @param allow_repl Controls whether the node.js REPL gets spawned when stdin
  * is an interactive terminal.
```

Comment:
`@return` is missing. Please add it.

Reply:
fixed in https://github.com/hpicgs/node/pull/80/commits/737c01556baa0b0cde76e98206cd187936a8ed57